libmei/atttypes.h

```diff
@@ -318,6 +318,12 @@ enum data_HEADSHAPE_list {
     HEADSHAPE_list_slash,
     HEADSHAPE_list_square,
     HEADSHAPE_list_x,
+    
+    /* The following SMuFL glyphs are not part of MEI's data.HEADSHAPE.list */
+    HEADSHAPE_list_circlex,
```

Comment:
This is problematic because the fill is auto generated. The proper way to fix it is to have the MEI data.HEADSHAPE.list extended accordingly. The issue should be raised on the music-encoding repository